ambry-cloud/src/main/java/com.github.ambry.cloud/VcrMetrics.java

```diff
@@ -42,6 +45,10 @@ public VcrMetrics(MetricRegistry registry) {
     blobDecryptionTime = registry.timer(MetricRegistry.name(CloudBlobStore.class, "BlobDecryptionTime"));
     blobUploadSkippedCount = registry.counter(MetricRegistry.name(CloudBlobStore.class, "BlobUploadSkippedCount"));
     updateTtlNotSetError = registry.counter(MetricRegistry.name(CloudBlobStore.class, "UpdateTtlNotSetError"));
+    addPartitionErrorCount = registry.counter(MetricRegistry.name(CloudBlobStore.class, "AddPartitionErrorCount"));
+    removePartitionErrorCount =
+        registry.counter(MetricRegistry.name(CloudBlobStore.class, "RemovePartitionErrorCount"));
+    tokenReloadWarnCount = registry.counter(MetricRegistry.name(CloudBlobStore.class, "TokenReloadWarnCount"));
```

Comment:
Instead of `CloudBlobStore.class`, I think here should be `CloudBackupManager.class`.